routers/books.js

```diff
@@ -103,14 +106,29 @@ bookRouter.put('/', (req, res) => {
  * @apiGroup Books
  * @apiVersion 0.1.0
  *
- *@apiParam {String} id Id of the Book being fetched.
+ *@apiParam {String} id Id of the Book being deleted.
  *
  * @apiSuccessExample {json} Success-Response:
- * 		HTTP/1.1 204 OK
+ * 		HTTP/1.1 204 No Content
```

Comment:
do we really have no content? We deleted a resource.

Reply:
so if we changed this to 200 what do we return then?

Reply:
nothing